client/components/pullRequests.js

```diff
@@ -10,7 +16,7 @@ class PullRequest extends Component {
     }
 
     calculateHours(date){
```

Comment:
As discussed, it makes more sense to have the server do these calculations which is then exposed through the API rather than the client side doing it.
But it can be done as a technical improvement.

Reply:
Although, you could argue because this is a presentation representation of the underlying 'creationDate' data, the calculation could be done in the client layer.
Just one to think about going forward when we add new attributes I think :)